docs/mca/quickstart.rst

```diff
@@ -0,0 +1,78 @@
+.. _label-mca-quickstart:
+
+Quick start: Run-time tuning Open MPI
+=======================================
+
+Although this section skips many details, it offers examples that will
+probably work in many environments.
+
+.. caution:: Note that this section is a "Quick start" |mdash| it does
+   not attempt to be comprehensive or describe how to build Open MPI
+   in all supported environments.  The examples below may therefore
+   not work exactly as shown in your environment.
```

Comment:
Need to update this text.

Reply:
What do you think should be changed?

Reply:
:...or describe how to build Open MPI in all supported environments."  Looks like that was copied from one of the other quick start sections -- but this section is not talking about building Open MPI.

Reply:
Oh derp, let me fix that

Reply:
Actually this should be fixed in all the quickstarts too